Makefile

```diff
@@ -73,7 +73,13 @@ pylint:
 	       --msg-template='{C}: {path}:{line}:{column}: {msg} ({symbol})'
 
 # TODO: add test and code quality checks for `examples`
-# TODO: add test for IPython notebooks in gammapy-extra
+
+gammapy-extra: 
+	git clone https://github.com/gammapy/gammapy-extra.git
 
 doc-show:
 	open docs/_build/html/index.html
+
+test-notebooks: gammapy-extra
```

Comment:
@cdeil Do you think it is preferable to run the test via ``python setup.py test -a '--nbval' -t <notebook>``?

If yes, this requires some extra effort due to
https://github.com/astropy/astropy/blob/master/astropy/tests/runner.py#L154